week2/index.js

```diff
@@ -1,65 +1,150 @@
+// require filesystem library because we are going to write and read files
 var fs = require('fs');
 
-var options = process.argv.slice(2);
+//this object will contain the todo's info and will be written to the todosFile 
+// I have chosen this to be a JSON because we can use the JSON proberities on it
+let todosFile = JSON.parse(fs.readFileSync('todo.json','UTF-8'));
+console.log(todosFile);
```

Comment:
Good for debugging, but did you mean to leave this in here for your submission?

This occurs no matter what the user enters, which is not in the spec.